### PR TITLE
Update delivery build documentation

### PR DESCRIPTION
**Pull request by peyerluk**
## Changelog

- 🎁 Add documentation for draft delivery builds
- 💎 Update existing configuration for builds and use 'delivery builds' as a name for this feature

### CODE REVIEW

File: content/guides/editor/publish-control/delivery/index.md [hunk 1]
```diff
@@ -113,3 +116,53 @@ Configure which data-records have a build button and build status in the publish
...
+  {
+    name: 'document.build.draft',
+    conditions: {
+      deliveryHandles: ['draftDelivery']
+    }
+  }
+]
+```
+
+The webhok workds the same as for delivery builds of type `publication` but there will be no publicationId in the event even if the document is published.
```
Comment (JSchenk8) [inline content/guides/editor/publish-control/delivery/index.md:151]: Typos: 
`The webhook works`

File: content/reference/project-config/deliveries.md [hunk 2]
```diff
@@ -5,14 +5,17 @@ menus:
...
-As of December 2020 this is used to enable the `Internal Document Links` feature.
+The deliveries can be used to configure these functionalities:
+- Enable the `Internal Document Links` feature.
+- Enable Publication and Draft Builds
 
 An example:
 ```js
 // deliveries is configured at projectConfig.deliveries
 deliveries: [
+  // Example which sports delivery build and internal links
```
Comment (JSchenk8) [inline content/reference/project-config/deliveries.md:18]: sports? Or supports?

File: content/guides/editor/publish-control/delivery/index.md (hunk 1 above)
Comment (looer) [inline content/guides/editor/publish-control/delivery/index.md:151]:
```suggestion
The webhook works the same as for delivery builds of type `publication` but there will be no publicationId in the event even if the document is published.
```

File: content/reference/project-config/deliveries.md [hunk 3]
```diff
@@ -57,7 +65,53 @@ This URL is then inserted into to `a` tags `href` attribute within the document.
...
 ### Caveats
+
 Since the `Internal Document Links` is specifically handy to link to not yet published documents you want to validate these links somewhere in your delivery to remove the ones pointing to not yet published documents.
 As this feature works on unpublished documents, Livingdocs cannot resolve the `routing` information from the [Routing System]({{< ref "/guides/organisation/routing-system.md" >}}) since this stores the information on publish which makes it unavailable before a document is published.
 
 To have the `Internal Document Links` feature be able to construct working `href` attributes, you need a routing system in your delivery that is able to redirect a user do a valid document URL based on the `document.id` and `document.projectId` only.
+
+## Delivery Builds for Publications
+
+A publication build adds a button to the Publish Panel which can be used to trigger a webhook event. This can be used to export publications to third party systems on demand or also to render previews externally and send a feedback with a link back to the Livingdocs Editor the user can click.
```
Comment (JSchenk8) [inline content/reference/project-config/deliveries.md:76]: *which the user can click

**Review by looer — APPROVED** [review]

File: content/guides/editor/publish-control/delivery/index.md [hunk 4]
```diff
@@ -1,14 +1,16 @@
 ---
-title: Publication Delivery
-description: Trigger deliveries and see delivery build status.
+title: Delivery Builds
+description: Trigger builds for specific deliveries and show the delivery build status in the Editor.
 weight: 6
 ---
 
 For infinite products like an e-paper there is usually a “build” stage involved. With the new Deliveries feature we allow a user to trigger a remote system to kick off a delivery build and report the status of the build back to Livingdocs. This allows an editor to get a better view on the infinite product life cycle. These are the features:
```
Comment (looer) [inline content/guides/editor/publish-control/delivery/index.md:7]:
```suggestion
For finite products like an e-paper there is usually a “build” stage involved. With the new Deliveries feature we allow a user to trigger a remote system to kick off a delivery build and report the status of the build back to Livingdocs. This allows an editor to get a better view on the infinite product life cycle. These are the features:
```

**Pull request merged**